### PR TITLE
MM-48753 - Calls: New GA cloud limits

### PR DESCRIPTION
**Pull request by cpoile**
#### Summary
- New cloud limits and text in the alert
- https://docs.google.com/document/d/1Tr8OBH4nPSN3SIMXWKY-_TMV-C0ht5zxmc2rLthZlLA/edit

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-48753
- 

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [x] Includes text changes and localization file updates
- [ ] Have tested against the 5 core themes to ensure consistency between them.

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: assets/base/i18n/en.json [hunk 1]
```diff
@@ -296,6 +296,7 @@
   "general_settings.display": "Display",
   "general_settings.help": "Help",
   "general_settings.notifications": "Notifications",
+  "general_settings.report_problem": "",
```
Comment (enahum) [inline assets/base/i18n/en.json:299]: I know is not from this PR but can you apply this?

```suggestion
  "general_settings.report_problem": "Report a problem",
```

**Review by enahum — APPROVED** [review]

File: app/products/calls/alerts.ts [hunk 2]
```diff
@@ -23,15 +24,21 @@ let recordingAlertLock = false;
...
     const title = intl.formatMessage({
-        id: 'mobile.calls_participant_limit_title',
-        defaultMessage: 'Participant limit reached',
+        id: 'mobile.calls_participant_limit_title_GA',
+        defaultMessage: 'This call is at capacity',
     });
-    const message = intl.formatMessage({
+    let message = intl.formatMessage({
         id: 'mobile.calls_limit_msg',
         defaultMessage: 'The maximum number of participants per call is {maxParticipants}. Contact your System Admin to increase the limit.',
```
Comment (streamer45) [inline app/products/calls/alerts.ts:34]: Interesting, does this mean we weren't showing anything special for Cloud before? It's not like the sysadmin could have done anything to raise limits.

**Review by streamer45 — APPROVED** [review]
Thanks :+1:

File: app/products/calls/alerts.ts (hunk 2 above)
Comment (cpoile) [reply app/products/calls/alerts.ts:34]: @streamer45 We were showing the generic max participants message, so correct: nothing special for cloud.

**Pull request merged**